README.md

```diff
@@ -19,6 +19,7 @@ NOTE: this project used to be called `nix-review`
 - GitHub integration:
   - post PR comments with results
   - approve or merge PRs (the last one requires maintainer permission)
+  - Show PR comments/reviews
```

Comment:
Might be also worth adding an example here: https://github.com/Mic92/nixpkgs-review/pull/107/files#diff-04c6e90faac2675aa89e2176d2eec7d8R150

Otherwise this feature may just remain hidden.